pkgs/desktops/maxx/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv, fetchurl
+, libX11, libXext, libXi, libXau, libXrender, libXft, libXmu, libSM, libXcomposite, libXfixes, libXpm
+, libXinerama, libXdamage, libICE, libXtst
+, fontconfig, pango, cairo, glib, libxml2, atk, gtk2, gdk_pixbuf, mesa_noglu, ncurses5 }:
+
+let
+  version = "Indy-1.0.0";
+
+  deps = [
+    libX11 libXext libXi libXau libXrender libXft libXmu libSM libXcomposite libXfixes libXpm
+    libXinerama libXdamage libICE libXtst
+    stdenv.cc.cc fontconfig pango cairo glib libxml2 atk gtk2 gdk_pixbuf mesa_noglu ncurses5
+  ];
+in stdenv.mkDerivation {
+  name = "MaXX-${version}";
+
+  srcs = [
+    (fetchurl {
+      url = "http://maxxinteractive.com/downloads/${version}/FEDORA/MaXX-${version}-NO-ARCH.tar.gz";
+      sha256 = "004ia6xl8y2hmbq7j98ppgiwffjc62224x9q6w5z17sjibs8xcvx";
+    })
+    (fetchurl {
+      url = "http://maxxinteractive.com/downloads/${version}/FEDORA/MaXX-${version}-x86_64.tar.gz";
+      sha256 = "134z7hinh01w43m6xiqgh939w5w79860g4kzsd911rfcl3z353av";
+    })
+  ];
+
+  buildPhase = ''
+    while IFS= read -r -d $'\0' i; do
+      if ! isELF "$i"; then continue; fi
+      patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$i" || true
+      patchelf --set-rpath "${stdenv.lib.makeLibraryPath deps}" "$i"
+    done < <(find "." -type f -print0)
+
+    substituteInPlace bin/adminterm \
+      --replace /bin/bash /bin/sh
```

Comment:
bash and sh are programs which interpret different languages, so you are potentially introducing a bug here. 

If the upstream program is really a sh program, then you should use ${stdenv.shell}, and inform upstream to fix their system.